src/config.rs

```diff
@@ -684,6 +684,7 @@ create_config! {
             "Enables unstable features. Only available on nightly channel";
     disable_all_formatting: bool, false, false, "Don't reformat anything";
     skip_children: bool, false, false, "Don't reformat out of line modules";
+    hide_parse_errors: bool, false, false, "Hide errors from the parser";
```

Comment:
Could we set this to `true` be default?

Reply:
I think that would mean we fail silently if there are parsing errors. If we do that, then I think we should at least report that there are parsing errors (I thought about collecting and reporting the parsing errors in all cases, rather than having this option, but I think then we would lose the colouring).